doc/project/tsc.rst

```diff
@@ -0,0 +1,218 @@
+  Zephyr Project.
+- Appointed TSC members can select an Alternate from the same organization.
+
+Elected TSC Members
++++++++++++++++++++
+
+Per the `Zephyr project charter`_, TSC members can nominate representatives from
+the technical community at the rate of no more than one per quarter.
+
+- Majority vote is required to confirm a candidate.
+- Once elected, a TSC member serves for 2 years.
+- Elected TSC members do not have the right to appoint an Alternate.
+- To ensure continuity of the TSC, at the end of the 2 year term, the TSC is
+  required to reconfirm the membership of elected members. If the elected member
+  declines a new term or if the TSC fails to reconfirm the term, the seat will
+  be open for new nominations.
+- If an elected TSC member resigns before the end of the 2 year term, their spot
+  will be open for new members outside of the quarterly nomination limit. The
+  elected member will serve a 2 year term.
+- The TSC has the right to terminate elected members who become inactive and are
+  not fulfilling the responsibilities of TSC members as described in this
+  document.
+- The number of elected members shall not exceed 20% of the total of appointed
+  members.
+- Existing TSC members who were elected before May 2024 shall be re-confirmed
+  after completing the 2 year term since they were first elected.
+
+Suspensions
++++++++++++
+
+As noted under Section 8b of the Project Charter, Voting rights for a
+representative who misses three consecutive meetings are subject to suspension
+and suspended representatives do not count towards the quorum requirement.
+
+A representative’s suspension will end and voting rights be restored at the start
+of the next attended meeting. The TSC enforces the suspension policy for voting
+members who miss three consecutive TSC weekly meetings.
+
+Multi-day meetings (F2F events) are counted as "one" meeting. The TSC voted on
+February 16, 2022 to discontinue default enforcement of the suspension policy.
+The TSC voted on January 18, 2023 to re-enact default enforcement of the
+suspension policy.
+
+Notice of suspension will be sent to representatives who miss three consecutive
+meetings, noting that rights will be restored upon next attendance of a TSC
+meeting.
+
+.. note::
+
+   As per Section 4b of the Project Charter, Platinum and Silver
+   Members may choose to opt out of a voting seat on the TSC.
+
+   Members who opt out and then wish to reclaim their seat later will have their
+   voting rights restored at the start of the second consecutive meeting
+   attended following notification to the TSC Chair.
+
+
+Voting
+++++++
+
+Voting in the Zephyr Project is defined under Section 8 of the Project Charter.
+
+Additional points of clarity / TSC interpretation have been added below:
+
+- The Governing Board may opt to update the Charter to include the below
+  refinements. Until then, additional clarifications (if/where needed) will be
```

Comment:
Since this sentence is part of the bullet list, what "the below refinements" target is unclear.

Reply:
Would it make sense to make this a paragraph instead of a bullet point, if the "below refinements" mean the bullet points below.

Reply:
this was addressed